test/com/google/enterprise/adaptor/opentext/OpentextAdaptorTest.java

```diff
@@ -153,6 +157,19 @@ public static void setUpClass() {
   @Rule
   public ExpectedException thrown = ExpectedException.none();
 
+  private Request newRequest(String id) {
```

Comment:
As these helpers suggest, the proxy isn't needed at all. You can just change RequestMock to implement Request, and do away with these.

Reply:
Done.